modules/apps/remote-app/remote-app-admin-web/src/main/java/com/liferay/remote/app/admin/web/internal/portlet/RemoteAppEntryPortlet.java

```diff
@@ -37,6 +46,25 @@ public void render(
 			RenderRequest renderRequest, RenderResponse renderResponse)
 		throws IOException {
 
+		ScriptData scriptData = new ScriptData();
+
+		scriptData.append(
+			null, "RemoteProtocolBridge.default()",
+			_remoteProtocolBridgeModuleName + " as RemoteProtocolBridge",
+			ScriptData.ModulesType.ES6);
+
+		StringWriter stringWriter = new StringWriter();
+
+		scriptData.writeTo(stringWriter);
+
+		StringBuffer stringBuffer = stringWriter.getBuffer();
```

Comment:
Do you need the buffer here? I think calling toString() with the writer should be enough.

Reply:
Probably not? Just copied from https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-taglib/frontend-taglib/src/test/java/com/liferay/frontend/taglib/servlet/taglib/ComponentTagTest.java#L144-L152 🤷